test/projectile-test.el

```diff
@@ -198,6 +198,13 @@
     (let ((projectile-indexing-method 'alien))
       (should (equal '("a/b/c" "a/d/e") (projectile-dir-files "/my/root/" "a/"))))))
 
+(ert-deftest projectile-test-git-submodule-command ()
```

Comment:
Took a good guess as to where in the file to place this new test. Let me know if you want it moved.